tests/ignite/engine/test_custom_events.py

```diff
@@ -414,6 +426,26 @@ def assert_(engine):
 
         assert num_calls[0] == 1
 
+        once = [1, 5]
```

Comment:
Maybe we can refactor these tests by adding `once` into `_test` as arg. Ideally, we can rewrite the whole test `test_once_event_filter_with_engine` using pytest parametrize

Reply:
Yes, sounds like a good idea. I will get started on the implementation